codeception.yml

```diff
@@ -15,7 +15,7 @@ extensions:
 modules:
     config:
         Db:
-            dsn: 'mysql:host=127.0.0.1;dbname=chopbox_testing'
-            user: 'root'
-            password: ''
+            dsn: 'mysql:host=127.0.0.1;dbname=chopbox'
+            user: 'homestead'
+            password: 'secret'
```

Comment:
Why do you have all this here?. You already have this configuration in your _bootstrap.php . Please remove everything you recently added here